library/core/src/main/java/com/google/android/exoplayer2/video/DolbyVisionConfig.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+package com.google.android.exoplayer2.video;
+import com.google.android.exoplayer2.Format;
+
+import com.google.android.exoplayer2.ParserException;
+import com.google.android.exoplayer2.util.ParsableByteArray;
+
+/**
+ * Dolby Vision configuration data.
+ */
+
+public final class DolbyVisionConfig {
+  public final int profile;
+  public final int level;
+  public boolean rpuPresent;
+  public boolean blpresent;
+  public boolean elpresent;
+
+  /**
+   * Parses Dolby Vision configuration data.
+   *
+   * @param data A {@link ParsableByteArray}, whose position is set to the start of the DoVi
+   *     configuration data to parse.
+   * @return A parsed representation of the DoVi configuration data.
+   * @throws ParserException If an error occurred parsing the data.
+   */
+  public static DolbyVisionConfig parse(ParsableByteArray data) throws ParserException {
```

Comment:
Perhaps this can return an RFC 6381-style codecs string with the profile and level and be moved to become a private static method in `AtomParsers`?

Reply:
We were thinking of reusing this even in case of TS extractors. Won't that be a problem if we move this as private static in  AtomParsers?

Reply:
Ah, in that case fine to leave. Is there a need to introduce the new DolbyVisionConfig type or can it just return the codecs string?

Reply:
makes sense now. Do you have a recommended file to put this as a function? In case if we add more config functions,  I am not sure if we may need a separate file in future?